scripts/compendium.js

```diff
@@ -6,26 +6,24 @@ const nextButton = document.getElementById('next-button');
 // 60vw + 35 for margin between entries
 const ENTRY_SCROLL_FACTOR = (document.documentElement.clientWidth * 0.6) + 35;
 
-let mostRecentEntry = entries[0];
-
-for (let entry of entries) {
-    if (getEntryDate(entry) > getEntryDate(mostRecentEntry)) {
-        mostRecentEntry = entry;
-    }
-}
-
-mostRecentEntry.scrollIntoView({ behavior: 'smooth' });
-
-function getEntryDate(entryElement) {
-    return Date.parse(entryElement.id.replace('entry-', ''));
+//checks to see if url specifies an entry
+if(window.location.href[window.location.href.length - 11] === '-'){
+    //goes away from set entry to first entry
+    entries[0].scrollIntoView({behavior: 'auto'});
+
+    //scrolls back to set entry
+    let currentEntry = document.getElementById('entry-' + window.location.href.substring(window.location.href.length - 10));
+    currentEntry.scrollIntoView();
```

Comment:
Since you _are_ using `scroll-behavior: smooth` in the css though, I don't think you even need this if statement. The behavior of scrolling to an ID defined in the URL hash is also affected by the `scroll-behavior` css property, so it's smooth by default.

Reply:
Oh, yup, just tried it without and it still works